pages/autosuggest/hide-entered-text-label.page.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useRef, useState } from 'react';
+
+import Autosuggest, { AutosuggestProps } from '~components/autosuggest';
+
+const empty = <span>Nothing found</span>;
+const options = [
+  { value: 'Option 0', tags: ['tag1', 'tag2'], filteringTags: ['bla', 'opt'], description: 'description1' },
+  { value: 'Option 1', labelTag: 'This is a label tag' },
+  { value: 'Option 2' },
+  { value: 'Option', description: 'description2' },
+];
+const enteredTextLabel = (value: string) => `Use: ${value}`;
+export default function AutosuggestPage() {
+  const [value, setValue] = useState('');
+  const [hasOptions, setHasOptions] = useState(true);
+  const [hasEmpty, setHasEmpty] = useState(true);
+  const [hideEnteredTextLabel, setHideEnteredTextLabel] = useState(false);
```

Comment:
nit: for settings it is better to use page context so that the configuration stays in the url and can be easily used in integ tests, see: https://github.com/cloudscape-design/components/blob/main/pages/autosuggest/async.page.tsx#L37

Reply:
Makes sense! I'll add the page context in the next PR update